unidist/core/backends/mpi/core/common.py

```diff
@@ -55,6 +56,19 @@ class Operation:
     CANCEL = 11
 
 
+operations_dict = dict(
+    (value, name)
+    for name, value in inspect.getmembers(Operation)
+    if name not in dir(type("dummy", (object,), {}))
+)
+
+
+def get_opp_name(opp: int):
```

Comment:
```suggestion
def get_op_name(op):
```
Let's not add type hints for now. Also, please add a docstring.